.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,19 @@
+name: Benchmarks
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  Benchmarks:
+    runs-on: ubuntu-latest
+    container: ghcr.io/notional-labs/cosmos
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.19' # The Go version to download (if necessary) and use.
+      - name: run benchmarks
+        run: make bench
```

Comment:
where is this being stored? how do we compare differences?

maybe go bencher is the right approach here

Reply:
oh, I think we should use both.  This only stores in github actions. 

I do not know how to configure gobencher, though.